static/app/views/starfish/modules/databaseModule/queries.js

```diff
@@ -87,6 +95,46 @@ export const getTopOperationsChart = (date_filters, interval) => {
   `;
 };
 
+export const useQueryTransactionByTPM = row => {
+  const pageFilter = usePageFilters();
+  const {startTime, endTime} = getDateFilters(pageFilter);
+  const dateFilters = `
+    greater(start_timestamp, fromUnixTimestamp(${startTime.unix()})) and
+    less(start_timestamp, fromUnixTimestamp(${endTime.unix()}))
+  `;
```

Comment:
nit: We already create the date filters in `QueryDetailBody`

Reply:
That's fair, but fwiw I do like the idea of using the page filters hook here, it makes this more "independant", you won't have pass in the page filters every time, cleans it up a bit.